src/test/java/hudson/scm/SubversionSCMTest.java

```diff
@@ -97,8 +97,8 @@ protected void setUp() throws Exception {
         super.setUp();
         repo = SVNURL.parseURIDecoded("http://subversion.tigris.org/svn/subclipse");
 
-        // during the test, don't pollute the global authentication cache.
-        SubversionSCM.CONFIG_DIR = createTmpDir().getAbsolutePath();
+        // during the test, don't pollute the user's configuration (esp. authentication cache).
+        System.setProperty(SubversionSCM.class.getName() + ".configDir", createTmpDir().getAbsolutePath());
```

Comment:
Really, in this point you define the environment variable, only that. When `SubversionSCM` is instantiated, it hopes that `SubversionSCM.CONFIG_DIR` have this value for the tests. [Reference](https://github.com/jenkinsci/subversion-plugin/blob/master/src/main/java/hudson/scm/SubversionSCM.java#L3149) happens.